modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -797,15 +797,17 @@ ADLAROMANCEStressUpdateBase::computeStressFinalize(const ADRankTwoTensor & plast
   _cell_dislocations[_qp] = _old_input_values[_cell_output_index] + _cell_dislocation_increment;
   _wall_dislocations[_qp] = _old_input_values[_wall_output_index] + _wall_dislocation_increment;
 
-  // Prevent the ROM from calculating and proceding with negative dislocations
-  if (_cell_dislocations[_qp] < 0.0 || _wall_dislocations[_qp] < 0.0)
+  // Prevent the ROM from calculating and proceeding with negative dislocations
+  if ((_cell_dislocations[_qp] < 0.0 || _wall_dislocations[_qp] < 0.0) && (_apply_strain))
```

Comment:
This is the only way that strain driven tests are going to pass when we do code to code comparison with the LANL rom.  This will let us do code to code comparison with the LANL version of rom.  These are strain driven tests.

Reply:
@dschwen will you review this again

Reply:
The extra parentheses around the apply_strain are not necessary